Release 4.8.1

awead opened the pull request:
## Changes

* sort a work's files in alpha order
* better messaging when draft works exist
* force UTF-8 encoding on extracted text files

mtribone commented:
@awead I say that if it visually matches what is currently in production, we are good to go. Otherwise I could tweak that layout all week!

awead commented:
@mtribone it doesn't. Here's what's currently in production:
![current](https://user-images.githubusercontent.com/312085/32189118-2ee47334-bd80-11e7-952e-00f512bb2e0c.png)
Note the bold-face field names.

mtribone commented:
They just need some `class="sr-only"` on them.

mtribone commented:
![screen shot 2017-10-30 at 2 48 06 pm](https://user-images.githubusercontent.com/4163828/32189546-670d482a-bd81-11e7-8ac4-bb48082e4440.png)

awead commented:
@mtribone ok, but my point was that we could make the same as the show view, which displays those field names instead of hiding them visually. Otherwise, we'd need to create separate renderers for each fields displayed.

mtribone commented:
@awead okay. so if you'd like to just use the same partial and have them be the same, go for it. Does creator show the bold term or just the other metadata?

awead commented:
@mtribone it's the depositor, not creator, so we'd probably have to make a new renderer for that. I'm open to suggestions!

olendorf commented:
looks fine to me

mtribone commented:
@awead @olendorf I'm good with this for now.

cam156 commented on spec/features/featured_work_spec.rb:
```diff
@@ -17,11 +17,13 @@
         expect(page).to have_content(file1.title[0])
+        click_link(file1.keyword.first)
       end
+      expect(page).to have_content(file1.title[0])
```
@awead curious why the second check of title

awead replied:
@cam156 line 24 clicks on the facet link and we want to make sure it shows the work. Previously, this was not working because the facet wasn't being correctly translated.

nicktiberi committed:
Sort a work's files in alphabetical order by default (#1046)

awead committed:
Brings CircleCI config insync with main (#1062)

awead committed:
WorkVersionDetailComponent (#1061)

Replaces in the in-view logic with a separate component to determine
messages displayed to the user. If the user can edit the work, and a
draft exists, then the message will give them a link to it.

If the user does not have edit rights, the message only appears if
another published version exists.

awead committed:
Force UTF-8 encoding for extracted text files (#1063)

Shrine is downloading extracted text derivatives as ASCII-8BIT, but we
need them to be UTF-8 so that they'll index properly. Because
MetadataListener is created the files, we can reasonably assume that
they'll almost always be UTF-8.

Several of the existing file helper classes were refactored and combined
into a new FileHelpers class for better organization in the spec tests.
